src/traces/box/attributes.js

```diff
@@ -255,5 +268,6 @@ module.exports = {
             'Do the hover effects highlight individual boxes ',
             'or sample points or both?'
         ].join(' ')
-    }
+    },
```

Comment:
:zap: that `,` 

Not sure why our linting didn't pick this up.

Reply:
I added this as I placed the width property at the end of this list the first time around. I then moved to the middle to a more appropriate spot but forgot to ⚡️  the `,`